docs/CHANGELOG.md

```diff
@@ -1,6 +1,31 @@
 # Change Log
 
-## [0.4](https://github.com/formatc1702/WireViz/tree/v0.4) (2024-05-12)
+## [0.4.1] (2024-07-13)
```

Comment:
Noticed too late that the link to the release is missing
```
## [0.4.1](https://github.com/formatc1702/WireViz/tree/v0.4) (2024-07-13)
```
but no biggie, can be fixed in next release.

Reply:
@formatc1702 wrote:
> Noticed too late that the link to the release is missing
> 
> ```
> ## [0.4.1](https://github.com/formatc1702/WireViz/tree/v0.4.1) (2024-07-13)
> ```

Remember not using the old URLs containing your personal user name `formatc1702`!

Another thing about the link: Now that you've created release pages for the version tags, should the version links in CHANGELOG point to the code tree for the tag (as today), or should they point to the release page for the tag?  https://github.com/wireviz/WireViz/releases/tag/v0.4.1

> 
> but no biggie, can be fixed in next release.

True. However, adding a minor commit to `master` fixing only such meta-information (not changing any file used by the code) long before creating a new release might also be considered, as far as I can understand, but I also can accept a point of view against that...